presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -151,6 +151,30 @@ public void testMySqlTinyint1()
         assertUpdate("DROP TABLE mysql_test_tinyint1");
     }
 
+    @Test
+    public void testCharTrailingWhitespace()
```

Comment:
`TrailingSpace` (not "whitespace" in general). AFAIK space (U+0020) has special meaning, all other whitespace does not.